packages/fetchai/contracts/erc1155/README.md

```diff
@@ -0,0 +1,21 @@
+# Fetch ERC1155 Contract
+
+## Description
+
+This contract package is used to interface with a Fetch Oracle Client contract, which requests a real-world value from a Fetch Oracle contract.
```

Comment:
This is not correct. The ERC1155 contract wraps a common standard, please link to it.